### PR TITLE
Gavin/hackathon 2022 kick prow

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

For hacking.

### CODE REVIEW

File: qa-tests-backend/README.md [hunk 1]
```diff
@@ -116,3 +116,4 @@ You will need to start another proxy:
  Or use the script provided by the deployment script:
 
 `deploy/{k8s,openshift}/central-deploy/central/scripts/port-forward.sh 8000`
+
```
Comment (BradLugo) [inline qa-tests-backend/README.md:119]: Test review

Comment (gavin-stackrox) [inline qa-tests-backend/README.md:119]: comment

**Review by BradLugo — CHANGES_REQUESTED** [review]

File: qa-tests-backend/README.md (hunk 1 above)
Comment (gavin-stackrox) [reply qa-tests-backend/README.md:119]: Looks good now

**Pull request closed without merging**